src/vrct/vrctFs/vrct.go

```diff
@@ -0,0 +1,307 @@
+package vrctFs
+
+import (
+	"fmt"
+	"gopkg.in/mgo.v2/bson"
+	"os"
+	"slices"
+	"sort"
+	"strings"
+)
+
+const virtualFSPathPrefix = "/tmp/spito-vrct/fs"
```

Comment:
I suggest changing the name of this constant to VIRTUAL_FS_PATH_PREFIX to indicate that it's a constant. Just like in C/C++ convention

Reply:
My IDE throw warning: `Use camel case instead of snake case`
I think (maybe I'm wrong) that if goland suggest constant to be camel case it is probably a good practice in go.